tests/01_crossing.spec.ts

```diff
@@ -54,21 +54,29 @@ test.describe("vtbag", () => {
 
 
   test('preserves animations and media timing', async ({ page }) => {
-    // todo: unable to get the video to play yet
```

Comment:
was able to get the video case to work and also added a check for CSS animation.